docs/providers/Analytics.tsx

```diff
@@ -22,6 +23,13 @@ export function AnalyticsProvider(props: AnalyticsProps) {
     };
   }, []);
 
+  useEffect(function didMount() {
+    events.on('routeChangeComplete', reportPageViewForGA4);
+    return function didUnmount() {
+      events.off('routeChangeComplete', reportPageViewForGA4);
+    };
+  }, []);
+
```

Comment:
The reports methods should be a part of the original `useEffect`, we should avoid two mount effects.